mappings/ctv3sctmap2/mappers.py

```diff
@@ -88,3 +89,39 @@ def ctv3_to_snomedct(ctv3_ids):
             )
 
     return records
+
+
+def snomedct_to_ctv3(snomedct_ids):
+    """Convert SNOMED CT Concept codes to CTV3 Concepts IDs."""
+    ctv3_ids = set(
+        CTV3Concept.objects.filter(
+            snomedct_mappings__sct_concept__active=True,
+            snomedct_mappings__sct_concept_id__in=snomedct_ids,
+            snomedct_mappings__map_status=True,
+            snomedct_mappings__is_assured=True,
+        ).values_list("pk", flat=True)
+    )
+
+    ctv3_id_to_snomedct_id = defaultdict(list)
+    for mapping in Mapping.objects.filter(
+        ctv3_concept_id__in=ctv3_ids, map_status=True, is_assured=True
+    ).select_related("ctv3_concept"):
+        ctv3_id_to_snomedct_id[mapping.ctv3_concept.read_code].append(
+            mapping.sct_concept_id
+        )
+
+    code_to_name = ctv3.lookup_names(ctv3_ids)
+
+    records = []
+    for ctv3_id in ctv3_ids:
+        records.append(
+            {
+                "id": ctv3_id,
+                "name": code_to_name[ctv3_id],
+                "active": True,
```

Comment:
CTV3 concepts can be in one of [four statuses](https://github.com/opensafely/opencodelists/blob/master/coding_systems/ctv3/models.py#L10-L13).  Suggest dropping the "active" key for now.

Reply:
Fixed in 8df5628